arrays.dd

```diff
@@ -218,9 +218,12 @@ $(H2 $(LNAME2 array-copying, Array Copying))
 ---------
 int[3] s;
 int[3] t;
+int[] d = new int[3];
```

Comment:
Isn't the preferred syntax `new int[](3)` nowadays, because it also support multi-dimensional arrays, like `new int[][](3, 5)`. That syntax also looks a lot as if you'd allocate a static array on the heap, but it allocates a dynamic array of length 3.